tools/rebasehelpers/util/git.go

```diff
@@ -59,6 +59,18 @@ func (c Commit) DeclaredUpstreamRepo() (string, error) {
 	return repo, nil
 }
 
+// HasVendoredCodeChanges verifies if the commit has any changes to Godeps/_workspace/
+// or vendor/ directories.
+func (c Commit) HasVendoredCodeChanges() bool {
+	for _, file := range c.Files {
+		if file.HasVendoredCodeChanges() {
+			return true
+		}
+	}
+	return false
+}
+
+// HasGodepsChanges verifies if the commit has any changes to Godeps/Godeps.json file.
```

Comment:
As I said, it's also being used elsewhere wrt to `Godeps/Godeps.json` - `GodepsReposChanged`, specifically. That's why I went with this name and most importantly added godoc, so it's clear what it does 😄